recipes/mongo-cxx-driver/all/conandata.yml

```diff
@@ -24,6 +27,13 @@ sources:
     url: "https://github.com/mongodb/mongo-cxx-driver/releases/download/r3.6.7/mongo-cxx-driver-r3.6.7.tar.gz"
     sha256: "2c58005d4fe46f1973352fba821f7bb37e818cefc922377ce979a9fd1bff38ac"
 patches:
+  "4.0.0":
+    - patch_file: "patches/4.0.0-0001-dirs.patch"
+      patch_description: "disable documentation features, fix directories"
+      patch_type: "conan"
+    - patch_file: "patches/4.0.0-0002-remove-abi-suffixes.patch"
```

Comment:
thanks for your contribution!
given that this is a new major version, and unless there's any other reasoning - I'd stop applying this patch and ensure that the library files are named as they are supposed to as intended by the upstream maintainers, and that the names are reflected in cpp_info instead